src/app/modules/portal-shared/card/card.component.html

```diff
@@ -0,0 +1,25 @@
+<div  #card (window:resize)="width=card.getBoundingClientRect().width">
```

Comment:
(window:resize) whay are we using this??

Reply:
Based on the size the text inside the card automatically take 3 dots